fluentlenium-core/src/main/java/org/fluentlenium/core/domain/FluentList.java

```diff
@@ -259,6 +260,16 @@ public String getText() {
         return null;
     }
 
+    @Override
+    public FluentList<FluentWebElement> find(String name) {
+        return find(name, new Filter[0]);
+    }
+
+    @Override
+    public FluentList<FluentWebElement> find(String name, Predicate<FluentWebElement>... predicates) {
+        return null;
```

Comment:
If it is incomplete why shall I merge this into live code?

Could you please create an issue what is not working w/ Java 8 and we can work on that.